lib/id3c/api/schemas.py

```diff
@@ -100,3 +100,43 @@
     ],
     "additionalProperties": False
 }
+
+POST_INCIDENT_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "sample_id": {
+            "type": "string",
+            "minLength": 8,
+            "maxLength": 8
+        },
+        "collection_id": {
+            "type": "string",
+            "minLength": 8,
+            "maxLength": 8
+        },
+        "incident_date": {
+            "type": "string",
+            "format": "date"
+        },
+        "failure_type": {
+            "type": "string"
+        },
+        "swab_type": {
+            "type": "string",
+            "enum": ["ans", "mtb", "np", "tiny", "unk", "none"]
+        },
+        "collection_matrix": {
+            "type": "string",
+            "enum": ["dry", "utm_vtm", "pbs", "none"]
+        },
+        "corrective_action": {
+            "type": "string",
+            "enum": ["discarded", "continued processing"]
+        }
+    },
+    "allOf": [
+        { "required":
+            [ "collection_id" ] }
+    ],
```

Comment:
Not a blocker, but I think `"required": ["collection_id"],` will accomplish the same thing.